corehq/apps/smsbillables/migrations/0010_add_index_to_log_id.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+
+        # Changing field 'SmsBillable.gateway_fee'
+        db.alter_column(u'smsbillables_smsbillable', 'gateway_fee_id', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['smsbillables.SmsGatewayFee'], null=True, on_delete=models.PROTECT))
+        # Adding index on 'SmsBillable', fields ['log_id']
+        db.create_index(u'smsbillables_smsbillable', ['log_id'])
+
+
+        # Changing field 'SmsBillable.usage_fee'
```

Comment:
did some digging to try and see if i could figure out why `smsbillables` is in this state. didn't find anything too fruitful, but tried a couple of times to rewind and rerun the `--auto` schemamigration and the same result happened. i think we'd be kicking the can further down the road if i manually edit the migration for the next person who tries to do a schemamigration.

here's a diff of the models from `0009` and `0010`: https://www.diffchecker.com/qug1ngxo . i actually think `0010` more accurately represents what's in the code. what if we throw this up on staging, run the migration and have it QA'd to make sure nothing is broken? then any future migrations won't have this issue